src/dmd/traits.d

```diff
@@ -1960,35 +1960,67 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
             d.inuse = 1;
         }
 
-        for (auto p = s; !p.isModule(); p = p.toParent())
+        /*
+         prepend the namespaces in the linked list `ns` to `es`.
```

Comment:
`Prepend` not `prepend`, because the sentence ends with a period it ought to commit to being a sentence.

Also, there's this vague undocumented `bool` return.

Reply:
If it's going to prepend ns to es, ns should be the first parameter.

Reply:
> Prepend not prepend

Fixed

> If it's going to prepend ns to es, ns should be the first paramete

I prefer having the parameter being mutated as the first parameter to be consistent with UFCS (unfortunately it doesn't work with local functions).